pheweb/serve/static/region_layouts.js

```diff
@@ -1057,3 +1057,10 @@ window.panel_layouts.genes = {
 if (window.browser == 'FINNGEN_QUANT') {
     window.panel_layouts.association.data_layers[2].fields = ["association:id", "association:chr", "association:position", "association:ref", "association:alt", "association:pvalue", "association:pvalue|neglog10_or_100", "association:beta", "association:sebeta", "association:rsid", "association:maf", "association:most_severe", "association:fin_enrichment", "association:INFO", "ld:state", "ld:isrefvar"]
 }
+
+if (window.browser == 'GBMA') {
```

Comment:
again thinking here could all these be configurations and no window.browser checking?